squidDrone/src/propulsion/motor_builder.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SRC_PROPULSION_MOTOR_BUILDER_HPP_
+#define SRC_PROPULSION_MOTOR_BUILDER_HPP_
+
+#include <memory>
+
+#ifndef UNIT_TEST
+#include "motor.hpp"
+#include "propulsion_hardware_config.hpp"
+#else
+#include "motor_mock.hpp"
+#include "propulsion_hardware_config_mock.hpp"
+#endif
+
+namespace propulsion {
+
+/**
+ * @brief A static class which only exists to build a correct Motor object
+ */
+class MotorBuilder {
+ public:
+  /// Ctor is deleted, because it should not be instantiated
```

Comment:
Is 'Ctor' as abbreviation for 'constructor' necessary? (Better explicit than implicit?)

Reply:
Fixed in [this commit](https://github.com/elheck/Squiddrone/pull/230/commits/e04970a97abe893523fc4550125cd271e221a661).